Makefile

```diff
@@ -41,3 +41,9 @@ release: format_code
 	git add .
 	git commit -m "Update to $(VERSION)"
 	git tag $(VERSION)
+
+install-binary:
+	./scripts/install.sh
```

Comment:
Would this need to be called from the makefile?

Reply:
Yea it was just for debug purpose, but the name is outdated. Will remove it👌

Reply:
done in a808975